.github/workflows/php.yaml

```diff
@@ -0,0 +1,104 @@
+name: PHP
+
+on:
+    pull_request:
+        branches:
+            - '3.5'
+            - '3.6'
+    push: null
+
+permissions: {}
+
+concurrency:
+    group: '${{ github.workflow }}-${{ github.ref }}'
+    cancel-in-progress: true
+
+jobs:
+    tests:
+        name: '${{ matrix.php }} / Symfony ${{ matrix.symfony_version }} / ${{ matrix.dependencies }} / ${{ matrix.os }}'
+        strategy:
+            matrix:
+                os:
+                    - 'ubuntu-22.04'
+                php:
+                    - '8.0'
+                    - '8.1'
+                    - '8.2'
+                    - '8.3'
+                dependencies:
+                    - 'lowest'
+                    - 'highest'
+                symfony_version:
+                    - '~5.4.9'
+                    - '~6.3.0'
+                include:
+                    - os: 'windows-2022'
+                      php: '8.1'
+                      dependencies: 'highest'
+                      symfony_version: '~6.3.0'
+                exclude:
+                    - os: 'ubuntu-22.04'
+                      php: '8.0'
+                      dependencies: 'lowest'
+                      symfony_version: '~6.3.0'
+                    - os: 'ubuntu-22.04'
+                      php: '8.0'
+                      dependencies: 'highest'
+                      symfony_version: '~6.3.0'
+        runs-on: ${{ matrix.os }}
+
+        steps:
+            -   uses: actions/checkout@v3
+
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: none
+
+            -   uses: ramsey/composer-install@v2
+                with:
+                    dependency-versions: ${{ matrix.dependencies }}
+                    composer-options: '--with=symfony/console:${{ matrix.symfony_version }}'
+
+            -   run: composer exec -- phpunit --testsuite EndToEnd
+
+            -   run: composer exec -- phpunit --testsuite Integration
+
+            -   run: composer exec -- phpunit --testsuite Unit
+
+    static_analysis:
+        name: Static analysis
+        strategy:
+            matrix:
+                include:
+                    - php: '8.1'
+                      symfony_version: '~6.3.0'
+                      dependencies: 'highest'
+        runs-on: ubuntu-22.04
+
+        steps:
+            -   uses: actions/checkout@v3
+
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php }}
+                    coverage: none
+
+            -   uses: ramsey/composer-install@v2
+                with:
+                    dependency-versions: ${{ matrix.dependencies }}
+                    composer-options: '--with=symfony/console:${{ matrix.symfony_version }}'
```

Comment:
It's already merged, but looks like this will install only `symfony/console` in expected version, not all `symfony` dependencies.

Reply:
Composer is smart enough to match versions of other Symfony pkgs

Reply:
No.
```
  - Locking symfony/config (v6.3.2)
  - Locking symfony/console (v5.4.28) 👈🏻 
  - Locking symfony/dependency-injection (v6.3.5)
  - Locking symfony/deprecation-contracts (v3.3.0)
  - Locking symfony/error-handler (v6.3.5)
  - Locking symfony/event-dispatcher (v6.3.2)
  - Locking symfony/event-dispatcher-contracts (v3.3.0)
  - Locking symfony/filesystem (v6.3.1)
  - Locking symfony/finder (v6.3.5)
  - Locking symfony/lock (v6.3.2)
  - Locking symfony/mailer (v6.3.5)
  - Locking symfony/mime (v6.3.5)
```
All others are at 6.3

Reply:
I never thought a 5.4 Symfony pkg is compatible with other 6.3 ones.